R/modeltests.R

```diff
@@ -354,6 +369,19 @@ evaluateRuns <- function(model, mydir, gitPath, compScen, email, mattermostToken
       write(runsNotStarted, myfile, append = TRUE)
       write(" ", myfile, append = TRUE)
     }
+
+    # Evaluate result of tests/testthat
+    logStatus <- NULL
+    try(logStatus <- readLines("../test-full.log", warn = FALSE))
+    if (is.null(logStatus)) {
+      testthatResult <- "Could not check for the results of TESTTHAT, test-full.log not found"
+    } else if (! any(grep("FAIL", logStatus))) {
+      testthatResult <- "`make test-full` did not run properly. Check test-full.log"
+    } else if (any(grep("FAIL 1", logStatus))) {
+      testthatResult <- "Some tests FAIL in `make test-full`. Check test-full.log"
+    } else {
+      # "make test-full reports all tests PASS"
```

Comment:
I would prefer something like `tail(grep("[ FAIL", logStatus, value = TRUE, fixed = TRUE))` to be appended to the log status, so you see directly how many fails exists etc.

And: Why `grep("Fail 1", logStatus)`? Does that work if you have more than one fail?

Reply:
You are right. I tested this with the output of `make test-coupled-slurm` (`test-coupled.log`), but its output seems to be different in terms of structure from the output of `make test-full-slurm`. I`ll adapt it.